src/protocol/TopicRouteData.h

```diff
@@ -122,11 +122,13 @@ class TopicRouteData {
   std::string selectBrokerAddr() {
     int bdSize = m_brokerDatas.size();
     if (bdSize > 0) {
+      std::srand(std::time(0) + bdSize);
```

Comment:
Don't set seed every time.

Reply:
I think setting seed in an init function somewhere is good.

Reply:
> 



> I think setting seed in an init function somewhere is good.

agree